openedx/core/lib/xblock_builtin/xblock_discussion/xblock_discussion.py

```diff
@@ -88,6 +94,49 @@ def django_user(self):
             return None
         return user_service._django_user  # pylint: disable=protected-access
 
+    @staticmethod
+    def vendor_js_dependencies():
+        """
+        Returns list of vendor JS files that this XBlock depends on.
+        """
+        return get_js_dependencies('discussion_vendor')
+
+    @staticmethod
+    def js_dependencies():
+        """
+        Returns list of JS files that this XBlock depends on.
+        """
+        return get_js_dependencies('discussion')
+
+    @staticmethod
+    def css_dependencies():
+        """
+        Returns list of CSS files that this XBlock depends on.
+        """
+        if get_language_bidi():
+            return ['css/inline-discussion-rtl.css']
+        else:
+            return ['css/inline-discussion.css']
+
+    def add_resource_urls(self, fragment):
+        """
+        Adds URLs for JS and CSS resources that this XBlock depends on to `fragment`.
+        """
+        # Head dependencies
+        for vendor_js_file in self.vendor_js_dependencies():
+            fragment.add_resource_url(static(vendor_js_file), "application/javascript", "head")
+
+        for css_file in self.css_dependencies():
+            fragment.add_css_url(static(css_file))
+
+        # Body dependencies
+        fragment.add_javascript(loader.load_unicode("static/js/mathjax_include.js"))
+
+        for js_file in self.js_dependencies():
+            fragment.add_javascript_url(static(js_file))
+
+        fragment.add_javascript(loader.load_unicode("static/js/discussion_classes.js"))
```

Comment:
@andy-armstrong Removed in https://github.com/edx/edx-platform/pull/13483/commits/724a839f77255346e911b8de2468d8ec5f4c68ea.